cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc

```diff
@@ -55,6 +55,10 @@ With this fix, the correct AWS product ARN for US and China regions are used.
 //added Kepler Update 2
 |The userinput:[--tarball] option in twistcli does not scan for compliance checks. Currently, only vulnerabilities are detected successfully.
 
+//CWP-47729
+|tt:[Fixed in 30.01.153]
```

Comment:
@divyabhushan in case you added this here. This issue was not fixed in update 1 and hence should be added here only after we go GA in update 2. 

I spoke to @rcaballeromx and we are moving it to the look ahead as a "Planned Fix"